src/components/sidebar.js

```diff
@@ -4,13 +4,14 @@ import { useLocation } from "@reach/router";
 
 import SmartLink from "./smartLink";
 
-const NavLink = ({ to, title, children }) => {
+const NavLink = ({ to, title, children, ...props }) => {
   const location = useLocation();
-
+  
   let className = "toc-item";
   if (location && location.pathname.indexOf(withPrefix(to)) === 0) {
     className += " toc-active";
   }
+  className += (props.className ? ' ' + props.className : '');
```

Comment:
How is this related to the rest of the PR? Is it used?

Reply:
Experimented with the sidebar and classes. Now you cant set classname on a navigation item but not related to this PR. Still like to leave it.